Dart/src/com/jetbrains/lang/dart/ide/errorTreeView/DartProblemsViewPanel.java

```diff
@@ -175,11 +179,6 @@ private void updateStatusBar() {
   }
 
   private static void addReanalyzeActions(@NotNull final DefaultActionGroup group) {
-    final AnAction reanalyzeAction = ActionManager.getInstance().getAction("Dart.Reanalyze");
```

Comment:
Removing this means that if multiple projects are open they all have to be reanalyzed when any one of them seems to be out of sync. Is that what you want?